pydocx/openxml/wml/object.py

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.models import XmlModel, XmlCollection
+from pydocx.openxml.vml.shape import Shape
+
+
+class Object(XmlModel):
```

Comment:
Make sense. I didn't realize that [wml](http://www.datypic.com/sc/ooxml/ns-w.html) is the same that wordprocessing.